tests/configuration.py

```diff
@@ -0,0 +1,193 @@
+
```

Comment:
Header file explaining how to build this would be great.

Reply:
i am planning to have a dummy file with dummy data that needs to be configured manually before running testbed,
later on, the class structure will be self explainatory along with some comments, but for now an explaination in comments will be usefull. i will update it